pkg/operator/ceph/cluster/watcher.go

```diff
@@ -193,3 +197,83 @@ func (c *clientCluster) getCephCluster() *cephv1.CephCluster {
 
 	return &clusterList.Items[0]
 }
+
+// each OSD nearfull status is checked
+// OSD capacity is increased if we get nearfull warning
+func increaseOSDsCapacity(cephStatusChecker *cephStatusChecker) {
+	osdUsage, err := cephclient.GetOSDUsage(cephStatusChecker.context, cephStatusChecker.clusterInfo)
+	if err != nil {
+		logger.Debugf("failed to get osd usage")
```

Comment:
```suggestion
		logger.Debugf("failed to get osd usage. %v", err)
```

Reply:
If this failed, we need to return. Continuing will cause a crash if we try to reference `osdUsage`.